api/src/main/java/net/akami/yggdrasil/api/data/YggdrasilData.java

```diff
@@ -0,0 +1,16 @@
+package net.akami.yggdrasil.api.data;
+
+import org.spongepowered.api.data.DataRegistration;
+
+public class YggdrasilData {
+
+    public static final DataRegistration<BoolData, BoolData.Immutable> PERSISTENT = DataRegistration.builder()
```

Comment:
Just wondering, is that constant used anywhere in the code? Looking through the files, I haven't seen it being used. Is it for later use?

Reply:
It is used by a spell coming soon.